apps/src/templates/sectionsRefresh/QuickAssignTableHelpers.jsx

```diff
@@ -80,10 +81,26 @@ function updateSectionCourse(updateCourse, course) {
       versions => versions.is_recommended
     )?.id;
   }
+
+  const courseVersion = courseVersions[courseVersionId];
+  const isStandaloneUnit = courseVersion.type === 'Unit';
+
+  let hasLessonExtras;
+  let hasTextToSpeech;
+
+  if (isStandaloneUnit) {
```

Comment:
Are these options not supported for non-standalone units?

Reply:
They are.  The issue is that once you choose "Stand alone Unit", the computer should know at that point if these two items are available.  For all other material, you need to have a chosen unit to know if they are available.  This is why this is set at this point for standalone units